lessons/module-1/js-classes.md

```diff
@@ -6,29 +6,33 @@ tags: js, introduction, constructor functions, this, classes, objects
 
 ## Learning Goals
 
-* Understand what `this` is and how it changes based on context
-* Understand what a class is and what object instances are
+- Understand what `this` is and how it changes based on context
+- Understand what a class is and what object instances are
 
-## Vocabulary
+## New Vocabulary
```

Comment:
Yes, I love this! Such a great way for students to segment their learning into more digestible bites!